src/main/scala/it/pagopa/interop/purposeprocess/service/RiskAnalysisService.scala

```diff
@@ -7,12 +7,30 @@ import it.pagopa.interop.tenantmanagement.client.model.TenantKind.{GSP, PA, PRIV
 import scala.io.Source
 import spray.json._
 
-object RiskAnalysisService {
+trait RiskAnalysisServiceSupplier {
+  def get(): RiskAnalysisService
+}
+
+object RiskAnalysisServiceSupplier extends RiskAnalysisServiceSupplier {
+  override def get(): RiskAnalysisService = new RiskAnalysisServiceImpl
+}
```

Comment:
The supplier is not required, let's pass the `Impl` as dependency at startup time
```suggestion
```